Loop/Views/MicrobolusView.swift

```diff
@@ -101,6 +113,15 @@ struct MicrobolusView: View {
             }
 
             if viewModel.microbolusesWithCOB {
+                Section(footer:
+                    Text("This is the minimum Microbolus size that will be delivered. Only if the Microbolus calculated is equal to or greater than this will a bolus be delivered.")
+                ) {
+                    Picker(selection: $viewModel.pickerMinimumBolusSizeIndex, label: Text("Minimum Bolus Size")) {
+                        ForEach(0 ..< viewModel.minimumBolusSizeValues.count) { index in Text(String(format: "%.2f", self.viewModel.minimumBolusSizeValues[index])).tag(index)
```

Comment:
Add units, please, so that it is clear that these are not basal minutes, but units of insulin. Like "0.1 U".
And add it to description.

Reply:
Latest version should fix these points